audresample/core/lib.py

```diff
@@ -10,13 +10,23 @@
 bin_path = os.path.join(root, 'bin')
 if platform.system() == 'Windows':  # pragma: no cover
     # for some reason we have to load soxr.dll first...
-    lib_path = os.path.join(bin_path, 'soxr.dll')
+    lib_path = os.path.join(bin_path, 'windows', 'soxr.dll')
```

Comment:
soxr should be linked statically into the shared libraries. Could you try if it runs without the soxr libraries?

Reply:
No, it's not working:

![image](https://user-images.githubusercontent.com/173624/214883209-bebc9148-103e-4473-b665-d5a59db7156b.png)

see https://github.com/audeering/audresample/actions/runs/4016760065/jobs/6900218726

Reply:
Oh wait, I forgot to remove the line.

Reply:
Did you use the new binary created in our internal CI?

Reply:
Also the `vcruntime140.dll` and the other dll should not be needed with the new binary since they were also linked statically (MT).

Reply:
I now added all the new binaries (even the M1 ones) and removed all other binary files.

It seems like the binary for MacOS Intel is not correct:

![image](https://user-images.githubusercontent.com/173624/214895430-e315e769-0647-4351-9da0-89103a8b8213.png)

Reply:
Good catch, seems I swapped M1 and Intel! ;)

Reply:
But is also shows the M1 binary is actually arm64 :)

Reply:
Great, it is working now.